css-transitions/styles.css

```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245 245 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+}
+
+.card-wrapper {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228 228 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+}
+
+.card > img {
+  width: 100%;
+}
+
+.card:hover {
+  transition: 0.5s;
+  transform: scale(1.1);
+}
+
+.custom-button:hover,
+.custom-button:active {
+  background-color: red;
+  transition: 0.5s;
```

Comment:
you want the transition to be on the element itself, not the one with the pseudo-class. otherwise the transition only works when the pseudoclass is applied.